src/models/TransactionProductModel.ts

```diff
@@ -10,11 +15,22 @@ TransactionProduct.init(
       type: DataTypes.INTEGER,
       autoIncrement: true,
       primaryKey: true
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['pending', 'reversed', 'closed'],
```

Comment:
values should be coming from the defined `TransactionType` enum, no raw string should be used here